src/foam/nanos/dig/DigWebAgent.java

```diff
@@ -210,13 +210,15 @@ public void put(FObject o, Detachable d) {
         System.err.println("objects selected: " + sink.getArray().size());
 
         if ( "json".equals(format) ) {
-          foam.lib.json.Outputter outputterJson = new foam.lib.json.Outputter();
+          foam.lib.json.Outputter outputterJson = new foam.lib.json.Outputter(OutputterMode.NETWORK);
           outputterJson.output(sink.getArray().toArray());
 
           if ( email.length != 0 && !email[0].equals("") && email[0] != null ) {
             output(x, outputterJson.toString());
           } else {
+            out.println("<textarea style=\"width:700;height:400;\" rows=10 cols=120>");
```

Comment:
Don't wrap in a textarea because, while this makes the output look better in the GUI, it will make it difficult for programs to parse, which is the primary use-case for DIG. Instead, set the HTTResponse content type to get it to display properly without the textarea wrapper.

Ex.: response.setContentType("application/json");